docs/api-testing.md

```diff
@@ -1,51 +1,78 @@
-  <div className="container">
-  <h3><a href="/api-testing/quickstart/">Quickstart</a></h3>
-  <p>Get up and running quickly with our API testing cloud.</p>
-  </div>
-</div><br/>
+<img src="/img/api-testing/API Testing Getting Started.png" width="600"/>
 <div className="box-wrapper" markdown="1">
-  <div className="box box1 card">
-    <div className="container">
-    <h2>Create an API Test</h2>
-    <p>Below are the methods to create a test.</p>
-    <ul>
-        <li><a href="/api-testing/quickstart/#create-test">HTTP Client</a></li>
-        <li><a href="/api-testing/build-from-spec">Import from OpenAPI</a></li>
-        <li><a href="/api-testing/import-postman-collection/">Import from Postman Collection</a></li>
-        <li><a href="/api-testing/import-postman-collection/#importing-a-har-file-rdc-and-vdc">Import from HAR file (including HAR files from RDC and VDC)</a></li>
-        <li><a href="/api-testing/composer">Test Composer (write test from scratch)</a></li>
-        <li><a href="/api-testing/import-export-tests">Import Legacy APIF Test</a></li>
-    </ul>
-    </div>
-  </div>
-  <div className="box box2 card">
-    <div className="container">
-    <h2>Sauce School</h2>
-    <p>Browse through our courses to learn more about API testing.</p>
-    <ul>
-        <li><a href="https://training.saucelabs.com/codelabs/Module1-APITesting/index.html?index=..%2F..apiTesting#0">API Testing Basics</a></li>
-        <li><a href="https://training.saucelabs.com/codelabs/Module2-APITesting/index.html?index=..%2F..apiTesting#0">Best Practices</a></li>
-        <li><a href="https://training.saucelabs.com/codelabs/Module3-APITesting/index.html?index=..%2F..apiTesting#0">Writing Your First API Test</a></li>
-    </ul>
-    </div>
-  </div>
-</div>
-
-<br/>
-<br/>
-
-:::info
-Looking for our on-premise (legacy) docs? See the [API Fortress documentation](https://docs.saucelabs.com/api-testing/on-prem/quick-start/).
-:::
+
+<div className="box box1 card">
+<div className="container">
+<h2>Contract Testing</h2>
+<p>
+</p>
+<p>
+Use the API Testing platform to generate contract tests from an OpenAPI specification file. When used with Piestry, our mocking tool, you can test both the consumer and the provider.
+
+</p>
+<ul>
+<li><a href="/api-testing/contract-testing">API Contract Testing</a></li>
+<li><a href="/api-testing/mocking">API Mocking with Piestry</a></li>
+</ul>
+</div>
+</div>
+
+<div className="box box2 card">
+<div className="container">
+
+<h2>Functional Testing</h2>
+<p></p>
+<p>
+Create functional tests from multiple locations:
+<ul>
+<li>
+<a href="/api-testing/quickstart">HTTP client</a></li>
+<li>
+<a href="/api-testing/import-postman-collection">Postman collection (in the HTTP client)</a></li>
+<li>
+<a href="/api-testing/build-from-spec">OpenAPI spec file or HAR file</a>
+</li>
+<li>
+<a href="/api-testing/import-postman-collection/#importing-a-har-file-from-an-rdc-job">HAR files captured during real device testing</a>
```

Comment:
link above is no longer valid, now we have both RDC and VDC. Replace with:
<a href="/api-testing/import-postman-collection/#importing-a-har-file-rdc-and-vdc">Import from HAR file (including HAR files from RDC and VDC)</a>